entrypoint.sh

```diff
@@ -114,6 +117,21 @@ if [ $skipDirs ];then
     SARIF_ARGS="$SARIF_ARGS --skip-dirs $i"
   done
 fi
+if [ $trivyIgnores ];then
+  for f in $(echo $trivyIgnores | tr "," "\n")
+  do
+    touch ./trivyignores
```

Comment:
Just curious why do we need to `touch` the file?

Reply:
@simar7 You are right, we don't. I've removed that line :+1: